src/Controls/tests/DeviceTests/Elements/Border/BorderTests.cs

```diff
@@ -110,11 +110,12 @@ public async Task DoesNotLeak()
 			});
 
 			await AssertionExtensions.WaitForGC(handlerReference, platformViewReference);
+
 			Assert.False(handlerReference.IsAlive, "Handler should not be alive!");
 			Assert.False(platformViewReference.IsAlive, "PlatformView should not be alive!");
 		}
 
-		/* Commented out for now until border issues are fixed
+#if IOS
```

Comment:
Should there be a comment linking to the issues that prevents it from running on the other platforms?